### PR TITLE
Philip/wal 10 sell p1 eligible fiat wallets activity

### PR DESCRIPTION
**Pull request by plondon**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4/src/redux/data/fiat/types.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { FiatType, RemoteDataType, SBTransactionsType } from 'core/types'
+
+import * as AT from './actionTypes'
+
+// state
+export type FiatStateType = {
+  [key in FiatType]?: {
```
Comment (plondon) [inline packages/blockchain-wallet-v4/src/redux/data/fiat/types.ts:7]: Change to WalletFiatType

File: packages/blockchain-wallet-v4/src/redux/data/fiat/selectors.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { FiatType } from 'core/types'
+import { RootState } from 'data/rootReducer'
+
+export const getFiatData = (currency: FiatType, state: RootState) => {
+  return state.dataPath.fiat[currency]
+}
+
+export const getTransactions = (currency: FiatType, state: RootState) => {
```
Comment (plondon) [inline packages/blockchain-wallet-v4/src/redux/data/fiat/selectors.ts:8]: Change to WalletFiatType

File: packages/blockchain-info-components/src/Fonts/Icomoon/fonts/icomoon.svg [hunk 3]
```diff
@@ -82,6 +82,8 @@
...
+<glyph unicode="&#xe94d;" glyph-name="eur" d="M0 667.429c0 161.583 130.989 292.571 292.571 292.571h438.857c161.584 0 292.571-130.989 292.571-292.571v-438.857c0-161.584-130.988-292.571-292.571-292.571h-438.857c-161.583 0-292.571 130.988-292.571 292.571v438.857zM645.84 533.687h-209.156c19.712 74.739 67.346 110.876 128.669 110.876 47.909 0 84.868-22.449 111.148-43.255l38.327 81.582c-40.517 33.947-90.891 53.11-149.475 53.11-116.352 0-204.229-74.464-231.058-202.313h-56.669l-21.627-58.309h70.631c-0.547-9.037-0.821-18.344-0.821-27.926 0-9.033 0.274-18.066 0.821-26.829h-49.004l-21.627-59.132h78.297c26.829-127.572 114.706-201.49 231.058-201.49 56.393 0 108.683 17.247 147.833 51.193l-38.875 81.584c-23.815-19.163-60.774-41.34-108.957-41.34-61.597 0-108.957 35.317-128.669 110.054h130.311l28.197 59.132h-167.541c-0.549 8.488-0.823 17.521-0.823 26.829 0 9.582 0.274 18.889 0.823 27.926h191.634l26.555 58.309z" />
```
Comment (milan-bc) [inline packages/blockchain-info-components/src/Fonts/Icomoon/fonts/icomoon.svg:86]: I'm just curious where usd will came?

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts [hunk 4]
```diff
@@ -2022,6 +2022,7 @@ type MessagesType = {
   'scenes.transaction.headertext.explainer.bch': 'Bitcoin Cash (BCH) is a fork of Bitcoin built for everday transactions.'
   'scenes.transaction.headertext.explainer.btc1': 'Bitcoin (BTC) is the original crypto and the internet’s digital currency.'
   'scenes.transaction.headertext.explainer.eth': 'Ethereum (ETH) is a currency and computing platform. Built for developers and apps.'
+  'scenes.transaction.headertext.explainer.fiat': 'Store {currency} on your wallet and use it to Buy Crypto.'
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:2025]: 👏

File: packages/blockchain-info-components/src/Fonts/Icomoon/fonts/icomoon.svg (hunk 3 above)
Comment (plondon) [reply packages/blockchain-info-components/src/Fonts/Icomoon/fonts/icomoon.svg:86]: I extract these as svgs from figma then use icomoon.app to generate a font

File: packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.tsx [hunk 5]
```diff
@@ -94,9 +84,10 @@ const Navigation = (props: OwnProps & Props) => {
       </LinkContainer>
       {values(
         mapObjIndexed(
-          (coin: SupportedCoinType, i) =>
+          (coin: SupportedCoinType | SupportedFiatType, i) =>
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.tsx:87]: this is used on 4 places we might make one common type and use it.

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.tsx:87]: Agree, maybe I call it `SupportedWalletCurrencyType` ?

File: packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceChart/CoinPerformance/selectors.ts [hunk 6]
```diff
@@ -8,7 +8,7 @@ export const getData = createDeepEqualSelector(
     selectors.components.priceChart.getTime,
     selectors.core.settings.getCurrency
   ],
-  (priceIndexSeriesR, priceChartTime, currencyR) => {
+  (priceIndexSeriesR, priceChartTime: '1day', currencyR) => {
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceChart/CoinPerformance/selectors.ts:11]: maybe to put it on constant

**Pull request merged**